website/docs/r/cloud_project_region_storage_presign.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "ovh"
+page_title: "OVH: cloud_project_region_storage_presign"
+sidebar_current: "docs-ovh-resource-cloud-project-region-storage-presign"
+description: |-
+  Generates a temporary presigned S3 URLs to download or upload an object.
+---
+
+# ovh_cloud_project_region_storage_presign
+
+Generates a temporary presigned S3 URLs to download or upload an object.
+
+## Example Usage
+
+```hcl
+resource "ovh_cloud_project_region_storage_presign" "presigned_url" {
+  service_name = "xxxxxxxxxxxxxxxxx"
+  region_name  = "GRA"
+  name         = "s3-bucket-name"
```

Comment:
to rename to `bucket_name` instead of just `name`?

Reply:
I agree 100 % that it should be `bucket_name` but I thought we wanted to stay as close as we could to the API so I used the same parameter name.

Reply:
Yes a provider should be the mirror of the API :-)
I understand :)